communication.md

```diff
@@ -191,3 +191,13 @@ Feedback is important to us. Knowing where we stand gives us clarity on
 what can be improved. We run period performance reviews based on peer
 feedback every three months. The primary tool are questionnaires, where
 we rate the work of our colleagues as well as our own.
+
+## Showing progress through heartbeats
+
+Showing progress is essential to keep the whole team informed and to spot projects that have gotten stuck. Read the chapter [Show Progress](https://basecamp.com/shapeup/3.4-chapter-13) from Shape up before continuing.
+
+We show the progress in the main GitHub issue of each project. The minimal progress update is an image of the current hill chart. We use [hillchart.app](https://hillchart.app/) to create and update them. We must copy an actual image into the issue, as this allows us to compare updates and see which scopes are moving and which are stuck.
+
+Feel free to add a couple of lines of text that give more context.
+
+Ideally, we should post an update every Friday. This schedule is not set in stone, though; different projects might require different update frequencies.
```

Comment:
let's set a reminder for this in Slack?

Reply:
I'm not sure where to put such a reminder so that it doesn't create interferences for the others; It's only us three that need to write updates. I would, therefore propose that we set reminders for ourselves. You can do so in you personal channel for example.

Reply:
👌 sgtm

Reply:
I didn't even know that it's possible. Works like a charm 🎉